gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -105,6 +105,31 @@ public GsonBuilder() {
   }
 
   /**
+   * Constructs a GsonBuilder instance from a Gson instance. The newly constructed GsonBuilder
+   * has the same configuration as the previously built Gson instance.
+   *
+   * @param gson the gson instance whose configuration should by applied to a new GsonBuilder.
+   */
+  public GsonBuilder(Gson gson) {
```

Comment:
probably a redundant API with newBuilder(). make this package-private?

Reply:
Good idea. Will do.

Reply:
8058ab76